examples/tree/plot_unveil_tree_structure.py

```diff
@@ -118,7 +118,7 @@
           % (node_id,
              sample_id,
              feature[node_id],
-             X_test[i, feature[node_id]],
+             X_test[sample_id, feature[node_id]],
```

Comment:
Good catch, thanks! While you are at it, can you please replace `X` by `X_test` in this [line](https://github.com/scikit-learn/scikit-learn/pull/8653/files#diff-591837f56eeded3a56f5869e2e414600R117)?